src/NativeScript/ObjC/Inheritance/ObjCClassBuilder.mm

```diff
@@ -30,13 +30,12 @@ @protocol TNSDerivedClass
 using namespace JSC;
 using namespace Metadata;
 
-static WTF::CString generateAnonymousSubclassName(const char* className) {
-    int i = 0;
-    WTF::CString anonymousName;
+static WTF::CString generateSafeSubclassName(const char* className) {
+    WTF::CString anonymousName(className);
 
-    do {
-        anonymousName = WTF::String::format("%s%d", className, ++i).utf8();
-    } while (objc_getClass(anonymousName.data()));
```

Comment:
Do we all agree that this version is clear enough ? If so, leave it as is :)
ping @fealebenpae @ivanbuhov